handbook/communication/rfcs/index.md

```diff
@@ -18,6 +18,7 @@ Each RFC has a status that is in the title of the RFC (e.g. "RFC 1 WIP: Title").
 | REVIEW | The RFC is ready to be reviewed. The RFC explicitly lists whose approvals are required and a requested timeline for those approvals. |
 | APPROVED | All comment threads are resolved and the RFC has been approved by all required approvers. Just because an RFC is approved doesn't mean it will definitely be implemented (e.g. priorities may change, or new information might be discovered during implementation or code review that causes a change in the plan). It is in the author's best interest to avoid surprises at code review time by ensuring the RFC has a sufficient level of detail and has approval from all relevant stakeholders. |
 | ABANDONED | There are no plans to move forward with this RFC. The particular reason is communicated in the metadata section of the RFC. For example, the RFC may have failed to get the necessary approvals, it may be been superseded by another RFC, priorities may have changed, or we may not have resources to work on this RFC in the foreseeable future. |
+| IN PROGRESS | This RFC is currently being worked on and has been migrated to a GitHub issue and broken down (as needed) into sub-issues.
```

Comment:
I am fine adding an IN PROGRESS status, but I don't think RFCs universally need to be translated to GitHub issues.

```suggestion
| IN PROGRESS | This RFC is currently being worked on.
```